docs/source/adding-yara-rules.rst

```diff
@@ -12,22 +12,74 @@ BinaryAlert includes a number of `custom YARA rules <https://github.com/airbnb/b
 
 Clone Rules From Other Projects
 -------------------------------
-BinaryAlert makes it easy to clone YARA rules from other open-source projects:
+BinaryAlert makes it easy to clone YARA rules from other projects:
 
 .. code-block:: bash
 
   $ ./manage.py clone_rules
 
-This will copy a subset of YARA rules from several :ref:`open-source collections <yara-credits>`.
-You can add more rule sources in `rules/clone_rules.py <https://github.com/airbnb/binaryalert/blob/master/rules/clone_rules.py>`_
+This will copy a subset of YARA rules from several default :ref:`open-source collections <yara-credits>` into the ``rules/`` folder.
+The cloned folder structure will mirror that of the remote repository.
 
-.. note:: We are working on a more expressive configuration for cloning subsets of rule repositories.
+.. note:: To ensure all upstream changes are copied (including deletions), the cloned folder structure for each repo will be deleted before cloning. For example, ``rules/github.com/Yara-Rules/rules.git`` will be deleted from your local filesystem before cloning from ``Yara-Rules``.
 
+Configuring Rule Sources
+........................
+
+You can configure the remote rule sources in `rules/rule_sources.json <https://github.com/airbnb/binaryalert/blob/master/rules/rule_sources.json>`_. Each rule source is defined by a git-cloneable ``url``, an optional list of filepaths to ``include``, and an optional list of filepaths to ``exclude``.
```

Comment:
maybe nit, but I _think_ `filepath` should be two words throughout

Reply:
Good catch - I'm so used to seeing `filepath` in variables and such

Reply:
yeah I only suggested that to be more human-readable (since these are docs, not code, and you're not referencing a variable, etc named `filepath`)